src/clarity-angular/datagrid/datagrid-items.ts

```diff
@@ -12,7 +12,16 @@ import {Items} from "./providers/items";
     selector: "[clrDgItems][clrDgItemsOf]",
 })
 export class DatagridItems implements OnChanges, DoCheck {
-    @Input("clrDgItemsOf") public rawItems: any[];
+    private _rawItems: any[];
+
+    public get rawItems(): any[] {
+        return this._rawItems;
+    }
+
+    @Input("clrDgItemsOf")
+    public set rawItems(value: any[]) {
+        this._rawItems = value;
+    }
```

Comment:
Yep. I had some logic in there but that was not needed later on. Thanks for catching this.